<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info_Dialog</class>
 <widget class="QDialog" name="Info_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>288</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Graph</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="to_time_radioButton_2">
         <property name="text">
          <string>to time</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="to_values_radioButton">
         <property name="text">
          <string>to values</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Range, sec.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="range_checkBox">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="range_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="auto_checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>auto</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="up_pushButton">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Number of charts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="number_spinBox"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Time interval, sec.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="time_interval_lineEdit">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_pushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="save_pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="info_label">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>to_values_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>number_spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_values_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>auto_checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_time_radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>to_values_radioButton</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_time_radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>number_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_values_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>auto_checkBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_time_radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>range_checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_time_radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>range_spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_time_radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>range_checkBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
